src/app/components/ItemPage/ItemHoldings.jsx

```diff
@@ -36,55 +36,45 @@ class ItemHoldings extends React.Component {
     const collapsed = !this.state.expanded;
 
     return (
-      <ul>
-        {
-          holdings.map((h, i) => {
-            let itemLink;
-            let itemDisplay = null;
-
-            // if (!h.available) {
-            //   itemLink = <span className="nypl-item-unavailable">{h.accessMessage}</span>;
-            // } else if (h.isElectronicResource) {
-            //   itemLink = <a href={h.url}>View Online</a>;
-            // } else {
-              // NOTE: This is using `this.props.bibId` but it is wrong. It should be the item ID.
-              // Currently, hitting the API with items is not working.
+      <table className="nypl-basic-table">
+        <tbody>
+          {
```

Comment:
There's no mention in the wireframe but this should really have headings. Even if it's simple, for clarity when presenting the data. `Locations`, `Call Number`, `Status`, `Message`, `Request`

But we can verify with Sean if he wants that there even as a starting point.